### ISSUE TITLE
How build SQLite3 for SQLCipher on windows10

### ISSUE THREAD

**Issue by ghost** [TryGhost/node-sqlite3#587]
can't built it on windows.I so sorry to can't find any example or information of it.

**Comment by eugirdor** [comment]
I have also tried and completely failed to get this building with SQLCipher on Windows.  Has anyone done it successfully?

**Comment by michael-ts** [comment]
Surely somebody has accomplished this so a pre-compiled binary can be added to the repository.  This appears to be a ridiculously complicated process to suffer through and I'm sure lots of people including myself would be very greatful.

**Comment by michael-ts** [comment]
Since I posted my last comment I used this [tutorial](https://github.com/justinclift/sqlitebrowser/wiki/Win64-setup-%E2%80%94-Compiling-SQLCipher) and managed to get SQLCipher itself compiled.

But when I try to actually use this library it seems to ignore my commands for specifying the path to SQLCipher, and when I try to created an encrypted database the database is not encrypted.  The output of the compilation process itself shows no evidence of sqlcipher being used instead of sqlite3.

I'm not exactly sure how I should be encoding windows paths in the arguments, and none of the ways I tried worked, so I tried doing it from cygwin without success using these commands:

```
export CPPFLAGS=-I/cygdrive/c/Users/michael/Documents/GitHub/sqlcipher
export CXXFLAGS=-I/cygdrive/c/Users/michael/Documents/GitHub/sqlcipher
export LDFLAGS=-L/c/Users/michael/Documents/GitHub/sqlcipher
npm install sqlite3 --build-from-source --sqlite3_libname=sqlcipher --sqlite3=/cygdrive/c/Users/michael/Documents/GitHub/sqlcipher --save

```

What's the proper way to tell npm a windows path if I do it from a Windows command prompt?
